crates/notary/server/config/gramine/Makefile

```diff
@@ -0,0 +1,62 @@
+# notary-server testing only
```

Comment:
is it possible to put this gramine folder in a `tee` parent folder instead of `config`? just to be more distinctive